main.py

```diff
@@ -26,22 +26,23 @@ def handle_error(e):
 @app.route("/", methods=["GET", "POST"])
 def upload():
     if request.method == "POST":
-        csvfile = request.files["csvfile"]
+        samples = request.files["samples"]
+        projects = request.files["projects"]
         # Do something with the uploaded CSV file...
-        csv_data = csvfile.stream.read().decode("utf-8")
-        samplesheet = generate_genomics_sheet(csv_data, request.form)
+        samples_data = samples.stream.read().decode("utf-8")
+        projects_data = projects.stream.read().decode("utf-8")
+        samplesheet, flowcell = generate_genomics_sheet(samples_data, projects_data, request.form)
         response = make_response(samplesheet)
         response.headers["Content-Type"] = "text/csv"
         response.headers[
             "Content-Disposition"
-        ] = f"attachment; filename=CTG_SampleSheet.csv"
+        ] = f"attachment; filename=CTG_SampleSheet_{flowcell}.csv"
```

Comment:
I think we should keep flowcell on one place, but make sure it does not have to be written manually twice by lab staff

Reply:
Also, this information is already in the samplesheet so it's unnecessary.

Reply:
I vote for `CTG_SampleSheet.csv`, unless someone can tell a good reason to have it with flowcell

Reply:
I vote for `CTG_samplesheet.csv` as well!

Reply:
I changed to what it was, the reason I thought it might be neat is that the lab workers will probably not remove old samplesheets and so we can expect that we get a number added by their browser instead like this ```CTG_SampleSheet(14).csv``` and then we would remove this useless variability with some variability that may be useful

Reply:
The alternative is to include it in the name, and exclude it from within the samplesheet.

Reply:
Let's discuss this afternoon!